src/executables/util_cs.c

```diff
@@ -4336,12 +4350,20 @@ flashback (UTIL_FUNCTION_ARG * arg)
 
   while (summary_entry == NULL)
     {
+      POLL_FD input_fd = { STDIN_FILENO, POLLIN | POLLPRI, 0 };
+
       printf ("Enter transaction id (press -1 to quit): ");
       fflush (stdout);
 
       if (poll (&input_fd, 1, timeout * 1000))
 	{
-	  scanf ("%d", &trid);
+	  if (!scanf ("%d", &trid))
```

Comment:
When `if` condition has  -1(EOF), does it work normally?
(Actually I don't know this mechanism)

Reply:
Thanks for feedback,

I didn't consider that users can input EOF with ctrl+D. 

I will apply your feedback